app/services/mapping/goodsshipment/ConsigneeBuilder.scala

```diff
@@ -19,28 +19,20 @@ import forms.declaration.{ConsigneeDetails, EntityDetails}
 import javax.inject.Inject
 import services.Countries.allCountries
 import services.mapping.ModifyingBuilder
-import services.mapping.goodsshipment.ConsigneeBuilder.createConsignee
-import uk.gov.hmrc.http.cache.client.CacheMap
+import services.mapping.goodsshipment.ConsigneeBuilder.{createConsignee, isDefined}
 import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment
 import wco.datamodel.wco.declaration_ds.dms._2._
 
 class ConsigneeBuilder @Inject()() extends ModifyingBuilder[ConsigneeDetails, GoodsShipment] {
 
   override def buildThenAdd(consigneeDetails: ConsigneeDetails, goodsShipment: GoodsShipment) =
-    if (ConsigneeBuilder.isDefined(consigneeDetails))
+    if (isDefined(consigneeDetails))
       goodsShipment.setConsignee(createConsignee(consigneeDetails.details))
 
 }
 
 object ConsigneeBuilder {
 
-  def build(implicit cacheMap: CacheMap): GoodsShipment.Consignee =
-    cacheMap
-      .getEntry[ConsigneeDetails](ConsigneeDetails.id)
-      .filter(isDefined)
-      .map(consigneeDetails => createConsignee(consigneeDetails.details))
-      .orNull
-
   private def isDefined(consigneeDetails: ConsigneeDetails): Boolean =
```

Comment:
We can move across these isDefined methods to the class now, they were only in the object because the old build method needed access to them statically